README.md

```diff
@@ -1,27 +1,87 @@
+This specification corresponds to a hubverse-style “compound modeling task” that includes the following fields: "reference_date", "location". Samples then capture dependence across the complementary set of task ids: “horizon”, “clade”.
+
+We note that sample IDs present in the output_type_id column of submissions are not necessarily inherent properties of how the samples are generated, as they can be changed post-hoc by a modeler. For example, some models may make nowcasts independently by horizon but the samples could be tied together either randomly or via some other correlation structure or secondary model to assign sample IDs that are consistent across horizons. As another example, some models may make forecasts that have joint dependence structure across locations as well as horizons. Sample IDs can be shared across locations as well, but this is not required for the submission to pass validation.
+
+While the hub requires predictive means to be submitted, to be included in the hub ensemble model, samples must be submitted and the mean forecast for the hub ensemble will be obtained as a summary of sample predictions.
+
+## Model evaluation challenges
+Several features of these data in particular make evaluations tricky. 
+
+1. Data for some model tasks may be partially observed at the time nowcasts and forecasts are made. The hub encourages teams to submit predictions of “true” underlying clade probabilities that will vary more or less smoothly, if sometimes steeply, over time. When some observations are partially observed at the time of nowcast submissions, it could be to the modeler’s advantage to predict a value that is close to the frequency observed at the time the forecast is made, thus deviating from the underlying (likely smooth) function the model would predict in the absence of data. To incentivize “honest” nowcasts that do not shift predictions for time-points with partial observations, we will only evaluate locations and dates for which no data have yet been reported at the time the clade list is generated in Monday. 
+<!-- TODO: check the above statement is true - is it the Monday date after which no reported data is considered, or a Wednesday? -->
+One implication of this decision is that different numbers of days may be evaluated for some locations when compared with others.
+
+2. The reference phylogenetic tree that defines clades changes over time. Nowcasts and forecasts will be evaluated against whatever sequence data is available 90 days after the deadline that a set of predictions were submitted for. Additionally, those sequences will be assigned a clade based on the reference tree that was used to generate the list of predicted clades on the Monday prior to the submission date. This means that new clades that emerge in the time since the predictions were made will still be classified as they would have been when predictions were made.
```

Comment:
> This means that new clades that emerge in the time since the predictions were made will still be classified as they would have been when predictions were made.

Should the beginning of this sentence be _This means that new **sequences** that emerge in the time since the predictions where made_?

Reply:
Perhaps we need to add something along the lines of what you are suggesting, but what we are talking about is different here. This sentence (as is) refers to the situation where the reference tree evolves so at the time of evaluation there are new __clades__ that were not present at the time the predictions were submitted. We don't expect modelers to be able to see into the future about which clades will exist in 90 days.

Reply:
Others may interpret differently, but my read of the sentence as written is that the [new] clades themselves would be classified as they would have been 90 days ago, which is confusing because they didn't exist 90 days ago. Isn't it the sequences themselves that are being classified [as a specific clade] as they would have been 90 days ago

Reply:
I see what you mean and agree with your logic. I made the update.